src/inc/warningcontrol.h

```diff
@@ -69,6 +69,7 @@
 
 #pragma warning(error   :4700)   // Local used w/o being initialized
 #pragma warning(disable :4706)   // assignment within conditional expression
+#pragma warning(disable :4768)   // __declspec attributes before linkage specification are ignored
```

Comment:
I have just realized we have the warning disabling at two places. One is here and the other is in the compileoptions.cmake:
https://github.com/dotnet/coreclr/blob/master/compileoptions.cmake#L94
It seems it would be better to have them at one place only.

Reply:
Should I move them all to here? There are a lot of warnings in warningcontrol.h, so it will be difficult to move them to compileoptions.cmake.

i.e. remove
`add_compile_options(/wd4960 /wd4961 /wd4603 /wd4627 /wd4838 /wd4456 /wd4457 /wd4458 /wd4459 /wd4091 /we4640)`
and add
```cpp
#pragma warning(disable :4960)
#pragma warning(disable :4961)
...
```

Reply:
Since CI is already green, and we need this patch for building on VS 15.5, I will merge this and open an issue for the change you requested @janvorli.

Is that fine?

Reply:
@ahsonkhan yes, that's perfectly fine.